### PR TITLE
Replace usage of cirq.Circuit.from_ops(...) with cirq.Circuit(...) in aqt,ion,linalg,neutral_atoms,optimizers,protocols

### PR DESCRIPTION
**Pull request by Strilanc**
Part of #1787

This should be nothing but basically a search-replace of `Circuit.from_ops(` with `Circuit(`, followed by formatting, followed by tiny manual tweaks.

### CODE REVIEW

File: cirq/optimizers/convert_to_cz_and_single_gates_test.py [hunk 1]
```diff
@@ -75,16 +75,16 @@ def _decompose_(self, qubits):
...
-    circuit = cirq.Circuit.from_ops(
+    circuit = cirq.Circuit(
         CompositeDummy()(q0),
         CompositeDummy2()(q0, q1),
     )
-    expected = cirq.Circuit.from_ops(
+    expected = cirq.Circuit(
         cirq.X(q0),
-        cirq.Y(q0) ** 0.5,
+        cirq.Y(q0)**0.5,
```
Comment (dstrain115) [inline cirq/optimizers/convert_to_cz_and_single_gates_test.py]: Any idea why the formatting changed here?  I think no spaces with exponents looks slightly better, but its weird that it changed suddenly.  Did the formatter version get upgraded?

File: cirq/optimizers/convert_to_cz_and_single_gates_test.py [hunk 2]
```diff
@@ -137,21 +133,17 @@ def test_passes_through_measurements():
...
-    circuit2 = cirq.Circuit.from_ops(
-        cirq.TwoQubitMatrixGate((np.array([[1, 0, 0, 0],
-                                           [0, 1, 0, 0],
-                                           [0, 0, 1, 0],
-                                           [0, 0, 0, 1j]]))).on(q0, q1)
-    )
+    circuit2 = cirq.Circuit(
+        cirq.TwoQubitMatrixGate((np.array([[1, 0, 0, 0], [0, 1, 0, 0],
+                                           [0, 0, 1, 0], [0, 0, 0,
+                                                          1j]]))).on(q0, q1))
```
Comment (dstrain115) [inline cirq/optimizers/convert_to_cz_and_single_gates_test.py]: This is really goofy formatting.

File: cirq/optimizers/convert_to_cz_and_single_gates_test.py (hunk 1 above)
Comment (Strilanc) [reply cirq/optimizers/convert_to_cz_and_single_gates_test.py]: It's just the first time these lines were hit by the autoformat. Exponentiation is done without spaces as a way of indicating it binds tighter than multiplication.

File: cirq/optimizers/convert_to_cz_and_single_gates_test.py (hunk 2 above)
Comment (Strilanc) [reply cirq/optimizers/convert_to_cz_and_single_gates_test.py]: Fixed

**Pull request merged**